src/index.js

```diff
@@ -439,14 +441,14 @@ let readScores = (workbook) => {
                 }
 
                 // Process trips.
-                // Add a "pass" object for each trip, including initial passes 
+                // Add a "pass" object for each trip, including initial passes
                 // (note that even incomplete initial passes get "pass" events.)
                 let skaterNum = ' '
-                let initCompleted = 'yes'
```

Comment:
In most programming languages, "init" is taken to mean "initialization" whereas in this case, it's the initial trip through the pack. Changed it for everyone's future sanity.